data_steward/tools/create_rdr_snapshot.py

```diff
@@ -5,12 +5,10 @@
 from datetime import datetime
 import logging
 
-from google.cloud import bigquery
-
 from cdr_cleaner import clean_cdr
 from cdr_cleaner.args_parser import add_kwargs_to_args
 from utils import auth
-from gcloud.bq import BigQueryClient
+from gcloud.bq import BigQueryClient, CopyJobConfig, WriteDisposition
```

Comment:
This works but seems a little odd to me.  `gcloud.bq` isn't defining or extending `CopyJobConfig` or `WriteDisposition`, so why are we importing them from there?  If someone were to remove their imports in `gcloud.bq` in the future, then this would break.  I would recommend importing them directly here as a Third party import (where you removed the `bigquery` import on line 8).
```python
from google.cloud.bigquery.job import CopyJobConfig, WriteDisposition
```

Reply:
Corrected

Reply:
was the change pushed?

Reply:
Yes